Makefile

```diff
@@ -202,7 +202,10 @@ tests:
 .PHONY: flake8
 flake8:
 	# E712 is not compatible with SQLAlchemy
-	find $(VALIDATE_PY_FOLDERS) -name \*.py | xargs flake8 \
+	find $(VALIDATE_PY_FOLDERS) \
+		-not \( -path "*/.build" -prune \) \
+		-not \( -path "*/node_modules" -prune \) \
```

Comment:
from where does those folders comes from?

Reply:
From non docker build in the admin folder.

Reply:
OK :-/